static/data.html

```diff
@@ -9,6 +9,7 @@
   <link href="css/bootstrap.min.css" rel="stylesheet">
   <link href="css/custom.css" rel="stylesheet">
   <link rel="icon" href="img/favicon.ico" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
```

Comment:
Will this stylesheet/font link be stable?

Reply:
It should, but just in case I will download it and place it into the css folder.

Reply:
I just added it in the last commit.